app/templates/admin/manageServiceLearningFaculty.html

```diff
@@ -111,37 +145,157 @@ <h5 class="ms-3">There are no approved courses for {{ term.description }}.</h5>
           {% endfor %}
         </tbody>
       </table>
-  {% endif %}
+      {% endif %}
+      </div>
+    </div>
 
+    {########################### Imported Table #################################}
+    <div class="tab-pane fade {{'active show' if courseID else ''}}" id="importedCourseTables" role="tabpanel">
+      <div class="container">
+        <div class="row">
+          <div class="col">
+            <h2>Imported</h2>
+          </div>
+        </div>
+      </div>
+
+  <!-- IMPORTED COURSES -->
+    {% if importedCourses | length == 0 %}
+    <h5 class="ms-3 pb-3">There are no imported courses for {{ term.description }}.</h5>
+    {% else %}
+    <table class="table table-striped border pb-5">
+      <thead>
+        <tr>
+          <th class="col-md-1" scope="col">Courses</th>
+          <th class="col-md-1" scope="col">Faculty</th>
+          <th class="col-md-1" scope="col">Action</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for course in importedCourses %}
+        <tr>
+          <td>{{course.courseName or (course.courseAbbreviation + " (name unspecified)")}}</td>
+          <td>{{course.instructors}}</td>
+          <td>
+            <select class="form-select" id="{{course.id}}" onchange='changeAction(this)'>
+              <option value="---" disabled selected>Select Action</option>
+              <option value="Modify">Edit</option>
+              <option value="View">View</option>
+              <option value="Print">Print</option>
+          </td>
+        </tr>
+        {% endfor %}
+      </tbody>
+    </table>
+    {% endif %}
+    </div>
+  </div>
+  {% set downloadAndSend = "disabled" if approvedCourses | length == 0 else "" %} 
+
+    <!-- Modal for editing imported courses -->
+    <div class="modal fade" id="alterModal">
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <h5 class="modal-header fw-bold" id="alterModalTitle"></h5>
+          <form enctype="multipart/form-data" role="form" method="POST">
+            <input type="hidden" id="alterCourseId" name="courseId">
+            <input type="hidden" id="alterTermId" name="termId" value="{{term}}">
+            <div class="modal-body"> 
+              <table class="table table-striped ml-4">
+                <tbody>
+                  <tr>
+                    <td class="fw-bold">Course Name</td>
+                    <td><input class="form-control" id="courseName" name="courseName"></td>
+                  </tr>
+                  <tr>
+                    <td class="fw-bold"> Course Abbreviation</td>
```

Comment:
Can you take away the space before Course or put one after Abbreviation?

Reply:
Fixed